scripts/medley/medley.cmd

```diff
@@ -1,3 +1,3 @@
-@echo off
-powershell medley.ps1 %*
-

```

Comment:
It's hard to see what changed here -- the GitHub compare shows no difference (The red 'before' and the green 'after' look identical), and the Lisp prc gives an error. And what changed doesn't seem to be obvious from the commit comments. ss

Reply:
This is just the result of the interim merges from master.